### ISSUE TITLE
Running MPOS pool, Some user cashouts never get to their payment address.

### ISSUE THREAD

**Issue by onecloudmanagement** [israelcoin/Isracoin#22]
I am running an MPOS pool, some user cashouts are not getting transaction ID and they are not getting their funds. When I use the isracoind sendtoaddress, the payments to those same addresses payments always go thru. 

any ideas?

**Comment by leofidus** [comment]
Since the problem apparently isn't with isracoin, you should ask other MPOS operators for help. You won't find much help on using MPOS here.

**Comment by onecloudmanagement** [comment]
I was told by TheSerapher that it is a coin Daemon issue. Can someone help me figure this out/

**Comment by fedaykinofdune** [comment]
If you're still having trouble, I will help you the best I can, having been through similar situations myself with a certain coin. It's possible that your daemon has problems with big transaction runs, in that case, try changing your MPOS configuration to use "Sendmany". In the case of sendmany, when successful, all payouts that were processed in the same batch will get the same TXID. If that does not help, then your wallet is having problems with transactions over a certain size, which is not unheard of. If that is the case try putting a very low limit on the amount of payments processed in a payout run in your MPOS config. If you're still having trouble, please contact me. I can be reached via the contact form from any one of my pools, just google "Fedaykin's Octocoin Pool" and do the contact form, I'll help as much as I can.

**Comment by fedaykinofdune** [comment]
I'd like to add a little bit, I guess I should have read some of your other issue reports first. Judging by your extreme orphan rate, you're experiencing some rather severe connectivity issues of some kind. If your daemon were connected properly with the network, you wouldn't be orphaning blocks like that, plus that explains issues with payout runs bailing out on you, failing to get a 200 OK or "halfway" processing, tricking MPOS into thinking your payouts succeeded when they didn't. Please get in touch with me before doing anything else, I will help you troubleshoot.

**Comment by onecloudmanagement** [comment]
Thanks for helping. I already have the sendmany configuration. What else can we look at. Network connections troubleshooting?

**Comment by fedaykinofdune** [comment]
Yes, definitely look at your network configuration. First a few questions..
Is your wallet and your pool on the same server? What kind of server are
you running on and how much memory and other resources do you have? What's
your connection speed?

If they're on the same server, that would preclude there being an actual
connection issue, but if they are that's definitely something you need to
look at. Also, if you're running everything on the same server and don't
have enough RAM you're definitely going to run into issues, especially with
the wallet, most wallets need 600+ MB of real memory (and not constantly
thrashing pagefile) to operate smoothly. If its running low on memory and
thrashing the pagefile constantly, that can really slow your payout runs
down, and cause some to bail out. Please contact me directly, so we're not
spamming these guys GitHub page trying to troubleshoot. Go to
http://denarius.fedaykin.us and use the contact form and get hold of me :)

**Comment by onecloudmanagement** [comment]
Wallet on same server. Should i move it to another server?

**Comment by fedaykinofdune** [comment]
Keep it on the same server, for best results. How much memory does it have?
An overworked system can lead to all kinds of errors. Also, on the payout
runs that failed, how many coins total was the system trying to send out?
Some coins have problems with large transactions.. Dont remember if I
mentioned that already, lack of sleep getting to me.. Please email me for
further troubleshooting. fedaykinofdune@gmail.com
